src/components/clusterDetail/FailedHostsWarning.tsx

```diff
@@ -47,6 +47,11 @@ const FailedHostsWarning: React.FC<FailedHostsWarningProps> = ({ cluster }) => {
       ' The cluster installation will time-out after 30 minutes if a master host does not reboot properly.';
   }
 
+  let failedHostsText = 'Error information for each worker host is available below.';
+  if (cluster.status !== 'finalizing') {
+    failedHostsText += ' Debug and reboot each host to try again.';
```

Comment:
```suggestion
    failedHostsText += ' Debug and reboot each failed worker to retry their installation.';
```

Reply:
resolved in 699c3cd7586fe1a80e1ea8ec0b2cbbd8024c9f34